portfolio/src/main/webapp/style.css

```diff
@@ -1,25 +1,61 @@
 #content {
   margin-left: auto;
   margin-right: auto;
-  width: 650px;
+  width: 90%;
 }
 
 #greeting-container {
   margin-top: 20px;
 }
 
 #me {
-  height: 500px;
-  margin-left: auto;
-  margin-right: auto;
+  height: 250px;
 }
 
 #images {
-    display: flex;
-    flex-wrap: wrap;
-    margin-left: 20%;
-    margin-right: 20%;
+  display: flex;
+  flex-wrap: wrap;
 }
 #images img {
-    height:250px
+  height: 250px;
+  width: auto;
+  margin: 10px;
+}
+
+header {
+  font-weight: bold;
+  font-size: 25px;
+}
+
+body {
+  font-family: 'Roboto Mono', monospace;
+}
+
+#landing-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+#background-container {
+  background-color: cornsilk;
+  padding: 5px;
+}
+
+#links-container {
+  display: flex;
+  justify-content: space-around;
+}
+
+#connect-container {
+  background-color: cornsilk;
+  align-items: center;
+  text-align: center;
```

Comment:
ditto

Reply:
Sorry, I alphabetized the CSS selectors instead. Thank you for catching that. I updated it to be the declarations.